resources/analysisTools/rnaseqworkflow/gtfParser.pl

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env perl
+use strict;
+
+#######################################################################################################################
+# AUTHOR: Nagarajan Paramasivam
+# DATE: 26th Feb 2021
+# EMAIL: n.paramasivam@dkfz.de
+
+# Made fore HIPO2 RNAseq workfow
+# Perl package containing functions to parse GTF files
+
+# An example gene line in the GTF file (tab separated)
+# chr1    HAVANA  gene    11869   14409   .       +       .       gene_id "ENSG00000223972.5"; gene_type "transcribed_unprocessed_pseudogene"; gene_name "DDX11L1"; level 2; hgnc_id "HGNC:37102"; havana_gene "OTTHUMG00000000961.2";
+#######################################################################################################################
+
+sub parseGTFFile{
```

Comment:
```suggestion
sub parseGTFFile ($$) {
```

... gives you early checks that the number of arguments is correct.